README.md

```diff
@@ -16,6 +17,36 @@ Here are links to good resources to get started:
 * [Debugging and Testing Agents](https://oxo.ostorlab.co/tutorials/debugging_agents)
 * [Understand OXO Internals](https://oxo.ostorlab.co/tutorials/life_of_a_scan)
 
+## Contributing
+
+To contribute to this project, follow these steps:
+1. Fork this repository.
+2. Clone your forked repository: `git clone <your_fork_url>`
+3. Navigate into your repository's directory: `cd <repository_directory>`
+4. Create a new branch: `git checkout -b <branch_name>`.
```

Comment:
mention the branch naming convention (fix/feature..)

Reply:
Those are linked to us.